src/events/eventToAction.js

```diff
@@ -154,14 +156,14 @@ export default (state: PerAccountState, event: $FlowFixMe): EventAction | null =
             : event,
       };
 
-    case EventTypes.restart:
-    case EventTypes.stream:
+    case 'restart':
```

Comment:
> Awkwardly this actually requires the cases to use string literals
> directly, rather than the named constants.  I prefer named constants
> for the sake of greppability... but when required to choose, checking
> for errors is more important.

Huh, yeah. Even this doesn't seem to work:

```js
    case (EventTypes.restart: 'restart'):
```

Shrug, ah, well.

Reply:
Yep. :-/